docs/source/cli-reference/other-commands.rst

```diff
@@ -5,16 +5,45 @@ Other available CLIs
 
 ----
 
-.. include::  /rst_snippets/scancode-reindex-licenses.rst
+.. include::  /rst_snippets/extract.rst
 
 ----
 
-.. include::  /rst_snippets/extract.rst
+.. include::  /rst_snippets/scancode-license-data.rst
+
+
+``--path`` Option:
+^^^^^^^^^^^^^^^^^^
+
+The ``--path`` option defines the directory where the license data will be stored.
+
+Here is an example of dumping license data with the ``--path DIR`` option::
+
+    scancode-license-data --path ~/licenses
+
+    Dumping license data to: /home/user/licenses
+    Done dumping #2465 licenses.
+
+Each of the licenses contains four files: '.LICENSE', '.html', '.json', and '.yml'.
+
+The dumped licenses directory look like this::
+
+    licenses/
```

Comment:
Could you also explain that this is available online at scancode-licensedb.aboutcode.org/ (generated from  https://github.com/aboutcode-org/scancode-licensedb which is refreshed automatically from scancode each day on new commits) and that someone can open this on their browser locally after using `scancode-license-data` by doing `see /path/index.html`

Reply:
@AyanSinhaMahapatra 
>  someone can open this on their browser locally after using scancode-license-data by doing see /path/index.html

I don't understand what does this mean?

Perhaps I can simply say `The license data is also avaliable at https://scancode-licensedb.aboutcode.org/` ?

Reply:
@chinyeungli do the following from a scancode-toolkit source checkout:

```
./configure --clean && ./configure --dev && source venv/bin/activate && scancode-reindex-licenses
mkdir licensedb
scancode-license-data --path licensedb
see licensedb/index.html
```
This will open the licenseDB webpages locally which you can browse (this is also generated by scancode-license-data), this is exactly the same as what we have at 
scancode-licensedb.aboutcode.org and how we generate those web pages in https://github.com/aboutcode-org/scancode-licensedb/blob/main/Makefile#L52 which is run automatically and updated daily.

So to summerize, I'd like a line in the docs to show that you can generate and browse this static licensedb website locally.